modules/__tests__/prefixer-test.js

```diff
@@ -1,119 +1,113 @@
-/* eslint-env jasmine */
-/* global jest */
-
-jest.dontMock('../prefixer.js');
-
-var originalDocumentCreateElement = document.createElement;
-var originalWindowGetComputedStyle = window.getComputedStyle;
-
 var browserPrefix = '';
 var mockStyle = {};
 
 describe('Prefixer', () => {
+  var exenv;
+
   beforeEach(() => {
     browserPrefix = '';
     mockStyle = {};
 
-    document.createElement = jest.genMockFunction().mockImplementation(() => {
+    sinon.stub(document, 'createElement', () => {
       return {style: mockStyle};
     });
-    window.getComputedStyle = jest.genMockFunction().mockImplementation(() => {
+    sinon.stub(window, 'getComputedStyle', () => {
       return [browserPrefix + 'transform'];
     });
 
-    jest.setMock('exenv', {
+    exenv = {
       canUseDOM: true,
       canUseEventListeners: true
-    });
+    };
   });
 
   afterEach(() => {
-    document.createElement = originalDocumentCreateElement;
-    window.getComputedStyle = originalWindowGetComputedStyle;
+    document.createElement.restore();
+    window.getComputedStyle.restore();
   });
 
   it('detects Firefox prefix', () => {
     browserPrefix = '-moz-';
-    var Prefixer = require('../prefixer.js');
-    expect(Prefixer.cssPrefix).toBe('-moz-');
-    expect(Prefixer.jsPrefix).toBe('Moz');
+    var Prefixer = require('inject!prefixer.js')({'exenv': exenv});
+    expect(Prefixer.cssPrefix).to.equal('-moz-');
```

Comment:
Future comment: You'll get more informative assertion failures with something like:

```js
expect(Prefixer).to.have.property('cssPrefix', '-moz-');
```

... but I don't want to clog up a straight porting PR with change suggestions ;)